package.json

```diff
@@ -2,7 +2,28 @@
   "name": "simple-server",
   "private": true,
   "dependencies": {
+    "@rails/webpacker": "^6.0.0-beta.7",
+    "bootstrap": "^5.0.2",
+    "css-loader": "^6.2.0",
+    "css-minimizer-webpack-plugin": "^3.0.2",
+    "mini-css-extract-plugin": "^2.1.0",
     "numeral": "^2.0.6",
-    "tablesort": "^5.2.0"
```

Comment:
Is tablesort still accessible via... sprockets(?) on existing pages?

Reply:
Hrm, good catch just saw this after merge...I'll take a look.

Reply:
Just verified, it still works.

I don't really know how all our old assets work TBH, I don't think they really use package.json at all for them.  Will be nice to remove sprockets altogether.